homeassistant/components/todoist/strings.json

```
@@ -20,6 +20,11 @@
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
   },
+  "exceptions": {
+    "section_invalid": {
+      "message": "Invalid section \"{section}\" within project \"{project}\""
```

Would something like this be more natural?
```suggestion
      "message": "Project \"{project}\" has no section \"{section}\""
```